src/main/java/core/basesyntax/operation/impl/BalanceOperation.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.operation.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.operation.OperationHandler;
+import java.util.Map;
+
+public class BalanceOperation implements OperationHandler {
+    @Override
+    public void handle(FruitTransaction transaction, Map<String, Integer> storage) {
```

Comment:
```suggestion
    public void handle(FruitTransaction transaction) {
```
Don't pass storage as a parameter. It should be a static map in the Storage class (fix it everywhere)